src/python/gudhi/sklearn/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .cover_complex import *
+
+__all__ = ["NerveComplex", "GraphInducedComplex", "MapperComplex"]
```

Comment:
(1) If you let this file empty, the "full" tests are still passing, and it will fix the tests without CGAL that are failing on the CI.
Because when no CGAL is found, NerveComplex is not compiled, so not found here.
What are the advantages in Python to list modules in `__all__` and `from .cover_complex import *` ?

(2) Note that I tested to merge this PR ( #403 ) and #737 and it works fine with and without CGAL this time. But the merge of `src/python/CMakeLists.txt` is not trivial at all...

# conclusion

We have to choose between option (1) or (2)

Reply:
I think we want #737 (or a variant of it) anyway, and the discussion of where exactly the new classes should be (gudhi.sklearn, gudhi.sklearn.cover_complex or elsewhere) becomes independent.